courses_api/serializers.py

```diff
@@ -60,12 +63,22 @@ class Meta:
             'thumbnails',
         )
 
+    def get_main_university(self, obj):
+        '''
+        The main university is just the first one, when ordered
+        using the order field.
+        '''
+        if not obj.first_university:
+            return None
+        return self.university_serializer_class(instance=obj.first_university).data
```

Comment:
This code makes two SQL queries where only one is required. I suggest:

    first_university = obj.get_first_university()
    if not first_university:
        return None
    return self.university_serializer_class(instance=first_university).data